### ISSUE TITLE
Can't make it run

### ISSUE THREAD

**Issue by JPOliveGit** [sampsyo/wideq#128]
in my case is pt-PT but it shows this, I'm new around here I searched in all issues but nothing came up like this.

joaooliveira@JPOlive-MBP wideq % python3 example.py -c PT -l pt-PT
Traceback (most recent call last):
  File "/Users/JPOlive/Desktop/wideq/example.py", line 3, in <module>
    import wideq
  File "/Users/JPOlive/Desktop/wideq/wideq/__init__.py", line 3, in <module>
    from .core import *  # noqa
  File "/Users/JPOlive/Desktop/wideq/wideq/core.py", line 9, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'

**Comment by sampsyo** [comment]
You will need to `pip install` the package to get the dependencies.

**Comment by JPOliveGit** [comment]
i was doing it wrong, now I obtained the two tokens, but when I tried to make the sensor

smartthinq_sensors:
  token: "refreshtoken"
  region: PT #(ISO 3166-1 alpha-2, e.g. PT)
  language: pt-PT #(ISO-639-1, e.g. pt)

the validation on the home assistant gives me this error:

 Invalid config for [sensor]: required key not provided @ data['platform']. Got None. (See /config/configuration.yaml, line 8).  

line 8:
line 9: sensor: !include include/sensor.yaml

line 8 is blank and if I remove line 9 it doesn't give me an error
